src/frontend/src/lib/i18n/en.json

```diff
@@ -230,6 +230,7 @@
 				"use_for_all_tokens": "Use for all tokens when receiving from wallets, users, or other apps that support this address format.",
 				"use_for_icrc_deposit": "Use for receiving ICRC-1 tokens in the ICP network, such as SNS, ck tokens, etc.",
 				"use_for_icp_deposit": "Use for ICP deposits from exchanges or other wallets that only support Account IDs.",
+				"your_private_eth_address": "Your private address for all ERC20 tokens.",
```

Comment:
The sentence is not exactly correct from a business perspective given that it's also the address for Ethereum itself and e.g. Sepolia. Is this expected?

Reply:
Hmm.. Let's tag people that decide this. @Stefan-Berger-DFINITY
ticket for reference: https://dfinity.atlassian.net/browse/OISY-203

Reply:
@peterpeterparker the sentence was suggested like that, but I see that it could be more precise. Do you have a good suggestion? Otherwise I will get back to the requester to ask for some improvement.

Reply:
"Your Ethereum address".

- I don't know what "private" means. I mean address are public keys.
- Not sure what that's wording for "your address that works on ethereum, sepolia, erc20 and future networks"

Reply:
But just my two cents, feel free to ignore.

Reply:
We can leave the sentence as it is, since it was asked for like that.